### DISCUSSION TITLE
historical_net_price and historical_net_reporting_period requirement

### DISCUSSION THREAD

**Discussion by mkoson-healthsparq** [CMSgov/price-transparency-guide#96] [Q&A] [answered]
For prescription-drugs, the schema notes that historical_net_price and historical_net_reporting_period are required fields. If this information is unavailable, can placeholder/""/null values be assigned?

**Comment by shaselton-usds** [comment, marked as answer]
@mkoson-healthsparq Both the `historical_net_price` and `historical_net_reporting_period` are required. 

Most likely an internal calculation will need to be made (and this will be different for different organizations) to determine the `historical_net_price` and then the `historical_net_reporting_period` can be populated based on the time series that was used during the calculation of the `historical_net_price`

**Reply by mkoson-healthsparq** [reply]
@shaselton-usds - the worry we have is that a lack of claims in any given 90 day period and subsequent small amount of historical information will result in fewer prescription-drugs objects than we would otherwise generate. From a downstream user perspective, it seems like `historical_net_price` and `historical_net_reporting_period` should be additive in informational value rather than forcing us to omit otherwise valid negotiated rates.
